src/music.ts

```diff
@@ -4,21 +4,22 @@
 // This class implements some standard music theory routines.
 
 import { Vex } from './vex';
+import { IMusicAccidental, IMusicNoteParts, IMusicNoteValue } from './types/common';
 
 export class Music {
-  static get NUM_TONES() {
+  static get NUM_TONES(): number {
     return 12;
   }
 
-  static get roots() {
+  static get roots(): string[] {
     return ['c', 'd', 'e', 'f', 'g', 'a', 'b'];
   }
 
-  static get root_values() {
+  static get root_values(): number[] {
```

Comment:
One option for a type alias here is `KeyValue`. This can also be used to compose `NoteValue` below (instead of using `number`.)

Reply:
In line with this I will change:
`root_indices()`

Reply:
I don't understand how a number array can a some KeyValue. What do you mean @0xfe?